functions/src/errorChecks.ts

```diff
@@ -1,69 +1,69 @@
 export function checkCVEValidity(ID: any): boolean {
```

Comment:
what's with the formatting changes in this file?

Reply:
I can fix it in a different PR